src/main/java/mods/eln/node/NodeBase.java

```diff
@@ -109,7 +109,7 @@ public void neighborBlockRead() {
 
             Block b = world.getBlockState(pos).getBlock();
             neighborOpaque |= 1 << direction.getInt();
-            if (isBlockWrappable(b, world, pos))
+            if (isBlockWrappable(b, world, pos)
```

Comment:
This is a syntax error. Try to get the files you're editing free of errors, so you can spot if this sort of thing happens.